internal/io/file/file.go

```diff
@@ -48,9 +54,29 @@ func Exists(filename string) (exists bool) {
 	return
 }
 
+func GetContents(filename string) string {
+	b, err := ioutil.ReadFile(filename)
+	if err != nil {
+		panic(err)
+	}
+
+	return string(b)
+}
+
+func OpenFile(filename string) *os.File {
+	f, err := os.Create(filename)
+	checkErr(err)
```

Comment:
>checkErr(err) [](http://example.com/codeflow?start=1&length=13)

can the app layer implementations of checkErr similar to how there can be multiple levels of a try/catch for exceptions?
If there's only a global error handler referenced by `checkErr` then any code that needs individualized error handling isn't going to use this wrapper.

Reply:
Added issue:

[https://github.com/microsoft/go-sqlcmd/issues/212](https://github.com/microsoft/go-sqlcmd/issues/212)